style.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: #3c3a3c;
+}
+
+.main {
+  display: flex;
+  background-color: #1c1a19;
+  background-image: url("Group.png");
+  background-repeat: no-repeat;
+  background-position: bottom -60px right 60px;
+  width: 375px;
+  height: 590px;
+  flex-flow: column;
+  justify-content: center;
+}
+
+.bars {
+  color: hsl(15, 5%, 85%);
+  padding-right: 90px;
+}
+
+.intro {
+  color: #ff6b00;
+  font-size: 52px;
+  font-family: 'Crete Round', serif;
+  margin: 10px 20px 10px 20px;
+  font-style: normal;
+  width: 250px;
+}
+
+.title {
+  color: #fff5e1;
+  font-size: 22px;
+  margin: 10px 20px 10px 20px;
+  font-family: 'Crete Round', serif;
+  width: 255px;
+}
+
+.logo {
+  font-size: 16px;
+  text-decoration: none;
+  color: #dbd8d7;
+  padding-left: 20px;
+}
+
+.text {
+  color: #fff;
+  font-size: 14px;
+  font-family: 'Poppins', sans-serif;
+  margin: 10px 20px 10px 20px;
+  width: 290px;
+}
+
+.contacts > ul > li > a {
+  color: #dbd8d7;
+}
+
+.contacts {
+  display: flex;
+  justify-content: inherit;
+  width: 180px;
+  height: 22px;
+  margin-left: 24px;
+  color: #dbd8d7;
+}
+
+.contacts > ul {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-left: 0;
+  gap: 16px;
+}
+
+.contacts > ul > li {
+  list-style-type: none;
+}
```

Comment:
- [ ] Please, kindly fix the width of your page and make it responsive as much as you can 😃. The hamburger icon should be fixed on the right. the same thing for the background image.  
- Your design:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/77190203/149025800-1a3af1d6-5c1d-4aa3-8a62-a87ddd8c2dfa.png">